_posts/2017-07-07-interview-with-kathryn-gronsbell.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title:  "Interview with Kathryn Gronsbell"
+date:   2017-06-13
```

Comment:
Good catch, thank you, my inability to remember this part ruins my life regularly.